### PR TITLE
design update

### PR DESCRIPTION
**Pull request by Famura17**

### CODE REVIEW

File: team1.html [hunk 1]
```diff
@@ -1,8 +1,43 @@
...
+  background-repeat: no-repeat;
+  background-attachment: fixed;  
+  background-size: cover;
+}
+</style>
 </head>
 <body>
-This is Team 1.
+
+<h2>Person details</h2>
```
Comment (shaziat) [inline team1.html:15]: can you change that to People

File: team1.html [hunk 2]
```diff
@@ -1,8 +1,43 @@
...
+	</tr>
+	<tr>
+	   <td><label for="crse"><b>Course Name:</label></td>
+  	   <td><input type="text" id="crse" name="crse" value="DevOPs Engineering"></td>
+  </table>
+</form>
+
+
 </body>
 </html>
```
Comment (shaziat) [inline team1.html]: requires a new line

**Review by shaziat — CHANGES_REQUESTED** [review]

File: team1.html (hunk 1 above)
Comment (Famura17) [reply team1.html:15]: This is how it should work.

File: team1.html (hunk 2 above)
Comment (Famura17) [reply team1.html]: line added

**Review by shaziat — APPROVED** [review]

**Pull request merged**